### PR TITLE
[elastic] Added new Elasticsearch plugin

### PR DESCRIPTION
**Pull request by Amitgb14**
Added Elasticsearch plugin to collect conf, log and some commands output.

### CODE REVIEW

**Comment by bmr-cymru** [comment]
Please follow the [contributor guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines) for patch formatting: the subject needs to be in `[<component>] change description` format.

**Comment by Amitgb14** [comment]
@bmr-cymru , It is okay now. :)

File: sos/plugins/elastic.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        ])
+
+        self.limit = self.get_option("log_size")
+        if self.get_option("all_logs"):
+            self.add_copy_spec("/var/log/elasticsearch/*",
+                               sizelimit=self.limit)
+        else:
+            self.add_copy_spec("/var/log/elasticsearch/elasticsearch.log",
+                               sizelimit=self.limit)
+
```
[the remark this answers is not in the document]
Comment (Amitgb14) [reply sos/plugins/elastic.py]: done...

**Comment by pmoravec** [comment]
+1 until you wish to add credits (like e.g. https://github.com/sosreport/sos/blob/master/sos/plugins/abrt.py#L1).

**Comment by Amitgb14** [comment]
@pmoravec , it's added now, thanks :)

File: sos/plugins/elastic.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018 Amit Ghadge <amitg.b14@gmail.com>
+
```
Comment (bmr-cymru) [inline sos/plugins/elastic.py]: You need to include the standard license header here: it is no longer the full GPL2 preamble, but a simple block of text that refers readers to the main LICENSE file:

```
# This file is part of the sos project: https://github.com/sosreport/sos
#
# This copyrighted material is made available to anyone wishing to use,
# modify, copy, or redistribute it subject to the terms and conditions of
# version 2 of the GNU General Public License.
#
# See the LICENSE file in the source distribution for further information.
```

This should appear beneath your own copyright declaration.

**Comment by Amitgb14** [comment]
Please, ignore the last commit, I'll send correct commit.

**Comment by bmr-cymru** [comment]
:+1:

**Comment by Amitgb14** [comment]
Thanks @bmr-cymru and @pmoravec .

**Pull request closed without merging**